Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:8.1.2-alpine
+
+ENV HOME=/usr/windbreaker
+
+RUN mkdir -p $HOME
+
+WORKDIR $HOME
+
+# install git
+RUN apk update && apk upgrade && \
+  apk add --no-cache git openssh
```

Comment:
Why include openssh? You can connect to docker containers without ssh. `docker exec -it <container_name> bash` (this applies to the other repos/prs as well)

Reply:
Will remove